src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml.cs

```diff
@@ -56,5 +70,27 @@ private void OnInstallVersionButtonDropDownClicked(object sender, RoutedEventArg
             OnShowContextMenuFromLeftClicked(sender, e);
         }
 
+        private void OnRequestShowFileDialog(object sender, EventArgs e)
```

Comment:
You can use `PackagePathEventArgs` instead of `EventArgs` to avoid a cast below.